src/Surfnet/ServiceProviderDashboard/Application/Console/CleanOldProductionEntitiesCommand.php

```diff
@@ -0,0 +1,104 @@
+<?php
+/**
+ * Copyright 2018 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\ServiceProviderDashboard\Application\Console;
+
+use Surfnet\ServiceProviderDashboard\Domain\Entity\Entity;
+use Surfnet\ServiceProviderDashboard\Domain\Repository\EntityRepository;
+use Surfnet\ServiceProviderDashboard\Infrastructure\Manage\Client\QueryClient;
+use Surfnet\ServiceProviderDashboard\Infrastructure\Manage\Exception\QueryServiceProviderException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class CleanOldProductionEntitiesCommand extends Command
+{
+    /**
+     * @var EntityRepository
+     */
+    private $entityRepository;
+    /**
+     * @var QueryClient
+     */
+    private $productionManageClient;
+
+    /**
+     * CleanOldProductionEntitiesCommand constructor.
+     * @param EntityRepository $entityRepository
+     * @param QueryClient $manageClient
+     */
+    public function __construct(EntityRepository $entityRepository, QueryClient $productionManageClient)
+    {
+        $this->entityRepository = $entityRepository;
+        $this->productionManageClient = $productionManageClient;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('app:clear-old-entities')
+            ->setDescription('Clears manage production entities not known to Manage.')
+            ->setHelp('This command allows you to clear old published production entities not known in to the production Manage.');
```

Comment:
Maybe increase the verbosity of this help message? Some essential parts are missing in my opinion. 

* We did not actually publish to manage prod, but simple send a publication request to the service desk.
* The prod published entity was stored in the databse (and not in Manage)
* Maybe inform the user that no actual Manage entities are to be removed from test/prod manage.

Reply:
I thought that was clear enough but If I'm able to avoid ambiguities I'm happy to do so.

Reply:
Thanks. In my opinion it's better to be absolutely clear instead of trusting on another person's domain knowledge.